zos_concepts/gdg_datasets/create_copy_submit/create_copy_submit.yml

```diff
@@ -0,0 +1,113 @@
+###############################################################################
+# © Copyright IBM Corporation 2024
+###############################################################################
+
+###############################################################################
+# This sample playbook demonstrates how to copy a directory to a PDS, edit a
+# member within the PDS and submit PDS member containing JCL.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories copy_edit_submit.yml
```

Comment:
This playbook performs no edits.

Reply:
Also need to use the correct playbook name create_copy_submit or whatever new name :)